### DISCUSSION TITLE
Which Pybricks version should I be using?

### DISCUSSION THREAD

**Discussion by FLL-Team-24277** [pybricks/support#1725] [General]
Last year we installed the firmware on our FLL robots from beta.pybricks.com. I don't remember why we needed the beta, but we did. We use VS Code and pybricksdev.exe to upload and run our programs. What are the main differences between the beta and release versions? Any reason we might want to choose one over the other for the upcoming FLL season?

**Comment by laurensvalk** [comment]
You can read about the two versions [here](https://pybricks.com/learn/getting-started/install-pybricks/#trying-new-beta-features-optional).

At this time, they are essentially the same, because there hasn't been a new beta release since the last stable release, `3.5.0`. You can see this in the [changelog](https://github.com/pybricks/pybricks-micropython/blob/master/CHANGELOG.md).

I would always suggest using the stable version, unless there is a beta feature that you absolutely need right away.

All features eventually come to the stable version. You probably used the beta version last year for the gyro functionality, which has since been included in the stable version also.
